UI/src/less/site/global.less

```diff
@@ -49,13 +49,23 @@ a {
   }
 
   .action-state {
+     display: inline-block;
+     padding: 0 10px;
+     border-radius: @border-radius-base;
+     font-size: @font-size-small;
+     line-height: @line-height-modifier * 6;
+     color: #fff;
+     background-color: @blue;
+   }
+
+  .controlTR {
     display: inline-block;
     padding: 0 10px;
     border-radius: @border-radius-base;
     font-size: @font-size-small;
     line-height: @line-height-modifier * 6;
     color: #fff;
-    background-color: @blue;
+    background-color: #0000ff;
```

Comment:
Is better put the color "#0000ff" in a variable at .less file like the other color above (@blue):